internal/entdb/client.go

```diff
@@ -94,6 +107,12 @@ func (c *EntClientConfig) NewMultiDriverDBClient(ctx context.Context, opts []ent
 		return nil, err
 	}
 
+	// Decorates the sql.Driver with entcache.Driver on the primaryDB
+	drv := entcache.NewDriver(
+		primaryDB,
```

Comment:
any reason not to include both?

```suggestion
		primaryDB,
		secondaryDB,
```

Reply:
Only the primary is used for reads, so didn’t think was needed on secondary. Although it won’t hurt it either.

Reply:
Fixed